cmd/api-mock/main.go

```diff
@@ -229,7 +229,7 @@ func (ase *AddonStatusEndpoint) ServeHTTP(w http.ResponseWriter, r *http.Request
 		fmt.Fprintln(w, string(respBytes))
 		log.Printf("%s %s:\n", r.URL.String(), r.Method)
 
-	case http.MethodPatch:
+	case http.MethodPost:
```

Comment:
Need to change the below logic to use the AddonID from the body and not the url.